libs/secure-sm/src/main/java/org/opensearch/secure_sm/SecureSM.java

```diff
@@ -208,6 +208,10 @@ private void debugThreadGroups(final ThreadGroup caller, final ThreadGroup targe
         System.err.println("access: target group=" + target);
     }
 
+    // thread context permissions
+
+    public static final Permission ACCESS_SYSTEM_THREAD_CONTEXT_PERMISSION = new ThreadContextPermission("markAsSystemContext");
```

Comment:
@cwperks I apologies for misleading you here, it seems like we do not refer to `SecureSM` directly from OpenSearch codebase (except on init part), keeping it under `ThreadContext` was right choice, could you please move it back? (my apologies again)

Reply:
Moved back to ThreadContext